### DISCUSSION TITLE
Able to define a sender format

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#420] [Feature Requests]
Currently user can choose between `John Wick - john at wick.com` or `John Wick - john(a)wick.com` format for the sender `John Wick <john@wick.com>`. 

It'd be helpful to allow users to define their own formats based on the `sender name` and the `sender address`, for example to match their filter setup. 

Moved from https://trello.com/c/5ctS2mwT/146-able-to-define-a-sender-format

**Comment by Metabaron1** [comment]
One usefull format to bypass spam filters based on sender domain: change sender from john@wick.com to john(a)wick(d)com

**Comment by xboxhaxorz** [comment]
Any format without the dash would be preferred
John Wick john at wick.com or John Wick john(a)wick.com or any other option, as it looks much cleaner and similar to the default gmail way

**Reply by nguyenkims** [reply]
For info, the dash sign (-) is to separate the *name* and *address* part in an email address.

**Reply by xboxhaxorz** [reply]
Yes i am aware, my other emails don't have the dash that are not from simple login

**Comment by jumpingbrowntrout** [comment]
Plus one for this feature request!!

Especially if using the feature "Include sender address in reverse-alias" it's not necessary to include anything other than the senders name as the senders name.

The present format is of "John Wick - John(a)wick.com" is far too messy in the inbox especially when the names and email addresses are long.
